Source/WebCore/platform/cocoa/PlaybackSessionModelMediaElement.mm

```diff
@@ -337,13 +342,13 @@
     else
         textTrack = &TextTrack::captionMenuOffItem();
 
-    m_mediaElement->setSelectedTextTrack(textTrack);
+    mediaElement->setSelectedTextTrack(textTrack);
 }
 
 void PlaybackSessionModelMediaElement::togglePictureInPicture()
 {
 #if ENABLE(VIDEO_PRESENTATION_MODE)
-    auto* element = dynamicDowncast<HTMLVideoElement>(*m_mediaElement);
+    RefPtr element = dynamicDowncast<HTMLVideoElement>(m_mediaElement);
```

Comment:
it seems to me that only this line was required to fix the crashes. Do we need all the above ?

Reply:
No, but it seemed like a good opportunity to make these mechanical changes.